src/triggers/index.js

```diff
@@ -441,12 +442,16 @@ export class ActionsTrigger {
     }
   };
 
-  loadPostRiver = async () => {
+  loadPostRiver = async (offset) => {
+    this.dispatch(setUIProgress('loadRiverInProgress', true));
+
     try {
-      let result = await this.client.subscriptions();
+      let result = await this.client.subscriptions(offset);
       this.dispatch(setPostsToRiver(result));
+      this.dispatch(setUIProgress('loadRiverInProgress', false));
```

Comment:
please don't!

just handle action triggered by `setPostsToRiver` in UI reducer. you don't need separate actions for this